packages/happo-target-react-native/native.js

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable */
+// this file isn't compiled, so we just ignore eslint here
```

Comment:
If it isn't too hard to only disable the rules that don't make sense for this file instead of disabling everything, that would be best.